src/cli/commands/info.js

```diff
@@ -72,7 +72,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const versions = result.versions;
   // $FlowFixMe
   result.versions = Object.keys(versions).sort(semver.compareLoose);
-  result.version = version || result.versions[result.versions.length - 1];
+  result.version = version || result['dist-tags'].latest;
```

Comment:
Are we always guaranteed to have `dist-tags` and `['dist-tags'].latest`?

Reply:
Who knows, it's entirely up to the npm server implementation :/ For now I think we can make this assumption, and revisit if needed.